File: src/main/java/com/zhongan/devpilot/completions/general/EditorUtils.java
//
// Source code recreated from a .class file by IntelliJ IDEA
// (powered by FernFlower decompiler)
//

package com.zhongan.devpilot.completions.general;

import com.intellij.openapi.application.ApplicationManager;
import com.intellij.openapi.editor.Editor;
import com.intellij.openapi.editor.EditorKind;

public class EditorUtils {
    public EditorUtils() {
    }

    public static boolean isMainEditor(Editor editor) {
        return editor.getEditorKind() == EditorKind.MAIN_EDITOR || ApplicationManager.getApplication().isUnitTestMode();
    }
}

File: src/main/java/com/zhongan/devpilot/completions/common/general/EditorUtils.kt
package com.zhongan.devpilot.completions.common.general

import com.intellij.openapi.application.ApplicationManager
import com.intellij.openapi.editor.Editor
import com.intellij.openapi.editor.EditorKind

object EditorUtils {

    @JvmStatic
    fun isMainEditor(editor: Editor): Boolean {
        return editor.editorKind == EditorKind.MAIN_EDITOR || ApplicationManager.getApplication().isUnitTestMode
    }
}
